README.md

```diff
@@ -1,41 +1,4 @@
-# Conditional URL Advertisements
```

Comment:
I'm not super comfortable having multiple translations in here since I'm not a native speaker and can't evaluate / vouch for the contribution. Could you change it back to just the one README file?

Reply:
As I said before, I can take care of the Chinese side